languages/go/oso.go

```diff
@@ -72,6 +72,16 @@ func (o *Oso) SetNotFoundError(notFoundError func() error) {
 	o.notFoundError = notFoundError
 }
 
+/*
+Set whether the Host accepts Expression types from Polar, or raises an error.
+
+	o, _ = oso.NewOso()
+	o.SetAcceptExpression(true)
+*/
+func (o *Oso) SetAcceptExpression(acceptExpression bool) {
```

Comment:
Hey! Sorry, one more quick change. The other libraries expose this method via the `Query` object instead. Please can we move it there to be consistent?